src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -136,6 +137,15 @@ private void changeCheckInServicePointId(UUID servicePointId) {
     write(representation, "checkinServicePointId", servicePointId);
   }
 
+  private void changeItemStatusToDeclaredLost(){
+
+    Item item = getItem();
+    if (item != null) {
+      item.changeStatus(ItemStatus.LOST);
+    }
```

Comment:
In other operations in the loan, e.g. `checkIn` the state that is changed is limited to the loan itself and the broader changes are handled outside the domain entity.

For example, for check in, the loan and item are changed (and stored) separately within the process adapter class.